src/table/__integ__/component-metrics.test.ts

```diff
@@ -12,7 +12,9 @@ declare const window: ExtendedWindow;
 
 class TableWithAnalyticsPageObject extends BasePageObject {
   async getComponentMetricsLog() {
-    const componentsLog = await this.browser.execute(() => window.__awsuiComponentlMetrics__);
+    const componentsLog: ExtendedWindow['__awsuiComponentlMetrics__'] = await this.browser.execute(
+      () => window.__awsuiComponentlMetrics__
+    );
```

Comment:
`this.browser.execute` is a polymorph function now: https://github.com/webdriverio/webdriverio/commit/bc5a7438cc8a402099ed9797adcc8ca4549012d9

Which does not play well with `__awsuiComponentlMetrics__` which is an `Array<any>`